examples/info_connect_client/main.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// Simple ConnectRPC client for testing the Info API
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+
+	infov1connect "github.com/ava-labs/avalanchego/proto/pb/info/v1/infov1connect"
+	v1 "github.com/ava-labs/avalanchego/proto/pb/info/v1"
+	"github.com/bufbuild/connect-go"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+func main() {
+	// Default to localhost, but allow overriding via command line
+	nodeURL := "http://localhost:9650"
+	if len(os.Args) > 1 {
+		nodeURL = os.Args[1]
+	}
+
+	// Create a client
+	client := infov1connect.NewInfoServiceClient(
+		http.DefaultClient,
+		nodeURL,
+	)
+
+	// Request node version
+	fmt.Println("Fetching node version...")
+	versionResp, err := client.GetNodeVersion(
```

Comment:
We should remove this file from the PR - since it seems to be used for manual testing purposes. If you want to test this PR you can follow some examples from our docs https://build.avax.network/docs/api-reference/info-api#infogetnodeversion that use `curl` from the command line to issue requests